IntegrationTests/tests_01_allocation_counters/test_01_resources/test_tiny_array_non_allocating_operations.swift

```diff
@@ -0,0 +1,30 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCertificates open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftCertificates project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftCertificates project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@_spi(IntegrationTests) import X509
+
+func run(identifier: String) {
+    measure(identifier: identifier) {
+        var counts = 0
+        counts += TinyArray(CollectionOfOne(1)).count
+        
+        do {
```

Comment:
Same note here on needing a loop.

Reply:
Because if you're wrong, you want to tell the difference between "the runtime just happened to fire an allocation from a background thread at this moment" and "the loop body actually allocates deterministically". The easiest way to tell that is to make any allocation that occurs in the loop body happen 1000 times, which we do by wrapping the test in a loop that iterates 1000 times.